### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by faarwa**
Last PR was too long ago so just did this again. This PR adds the footer with some social media links pulled from the data.

Screen Shot 2020-05-28 at 1 42 03 AM

For the SVGs I just copied and pasted from the files, not sure if that's the best way or if I should import the files.

### CODE REVIEW

File: gatsby-site/src/components/SunriseFooter.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            <ImageStyle>
+              {iconForSocialMediaType(socialMediaType)}
+            </ImageStyle>
+          </ImageCircle>
+      </div>
+    );
+  }
+
+  return <Container>
+    <Button label={`Donate to Sunrise ${hubName}`} href="https://secure.actblue.com/donate/sunrisemvmt" />
```
Comment (faarwa) [inline gatsby-site/src/components/SunriseFooter.js:110]: Ah I think I have to add padding under the button according to Figma...can do that tomorrow

File: gatsby-site/src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from "react"
```
Comment (lukewestby) [inline gatsby-site/src/components/Button.js:1]: 👍 good call extracting the button

File: gatsby-site/src/components/SunriseFooter.js (hunk 1 above)
Comment (lukewestby) [reply gatsby-site/src/components/SunriseFooter.js:110]: Everything else looks good so if you want to merge this without the padding we can come back to it in another PR

**Review by lukewestby — APPROVED** [review]
nice!

**Pull request merged**